### PR TITLE
5 add webpack

### PR DESCRIPTION
**Pull request by menglabs**
Add webpack into dv-web

### CODE REVIEW

File: app/public/index.html [hunk 1]
```diff
@@ -8,8 +8,8 @@
       manifest.json provides metadata used when your web app is added to the
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="manifest" href="manifest.json">
+    <link rel="shortcut icon" href="favicon.ico">
     <!--
       Notice the use of %PUBLIC_URL% in the tags above.
```
Comment (yiyidai) [inline app/public/index.html]: Why did you remove this dynamic url? Are the comments below still needed?

File: app/.babelrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["env", "react"]
+}
```
Comment (xiaoxipang) [inline app/.babelrc]: I'm not fully sure about this. But I remember usually we can config the babel inside the webpack.

**Review by xiaoxipang — COMMENTED** [review]
This is great!

File: app/.babelrc (hunk 2 above)
Comment (xiaoxipang) [reply app/.babelrc]: To do so, https://github.com/babel/babel-loader

Comment (xiaoxipang) [reply app/.babelrc]: They do mention you can combine babelrc with webpack or not.

Comment (menglabs) [reply app/.babelrc]: Let me try it tonight

Comment (menglabs) [reply app/.babelrc]: Yes, it is able to config in webpack.config.js

**Review by yiyidai — CHANGES_REQUESTED** [review]

File: app/public/index.html (hunk 1 above)
Comment (menglabs) [reply app/public/index.html]: Aha, finally I saw your comment! 
Webpack would not recognize this dynamic format, so I delete it.
You are right! The comments related to it should not be there, but this file would not be here anymore after tomorrow.

Comment (yiyidai) [reply app/public/index.html]: Thanks for addressing the comment from a "peanut gallery"  :p
